enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -677,6 +701,12 @@ func (mc *MigrationCache) Start() error {
 		mc.copyDataInBackground()
 		return nil
 	})
+	if mc.copyChanFullWarningInterval > 0 {
+		mc.eg.Go(func() error {
+			mc.logCopyChanFullInBackground()
+			return nil
```

Comment:
if the above function is returning error, you can just do return mc.logCopyChanFullInBackground

Reply:
Ack